phys/module_diagnostics_driver.F

```diff
@@ -62,7 +62,8 @@ SUBROUTINE diagnostics_driver ( grid, config_flags,                  &
                                    model_config_rec
 
       USE module_streams
-      USE module_utility, ONLY : WRFU_Time 
+!      USE module_utility, ONLY : WRFU_Time, WRFU_TimeInterval 
```

Comment:
@twjuliano 
Tim,
For code cleanliness, let us just remove the commented out code.

Reply:
@davegill I reverted back to what was originally there since we no longer need this code in the diagnostics driver